src/core/crypto.c

```diff
@@ -387,6 +389,48 @@ uint16_t Crypto_Calc_CRC16(uint8_t *data, int size)
     return crc;
 }
 
+uint8_t gf_mul(uint8_t a, uint8_t b) 
+{
+    if (a == 0 || b == 0) 
+    {
+        return 0;
+    }
+    else
+    {
+        return gf_exp[(gf_log[a] + gf_log[b]) % (GF_SIZE - 1)];
+    }
+}
+
+uint16_t Crypto_Calc_FHECF(uint8_t *data)
```

Comment:
Can we add a reference to the relevant document and section where is is captured?
I think it's CCSDS 732.0-B-4 (AOS Space Data Link Protocol) Section 4.1.2.6
May also be good to write out FHECF somewhere as Frame Header Error Control Field.